src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCarRent = 40;
+  const totalCarRent = days * dailyCarRent;
+
+  return days >= 7
+    ? totalCarRent - 50
+    : days >= 3
+      ? totalCarRent - 20
+      : totalCarRent;
```

Comment:
Don't write nested ternary operators.
This way it wirks, but extremely hard to read.
Replace with `if` structure, and don't forget to read checklist.

Reply:
ternary replaced with if else block.